ail_vru_uc1_scenario/bridge.yml

```diff
@@ -131,10 +139,18 @@ topics:
     type: autoware_msgs/msg/ControlCommandStamped
     queue_size: 1
 
+  - topic: /guidance/ctrl_cmd
+    type: autoware_msgs/msg/ControlCommandStamped
+    queue_size: 1
+
   - topic: /guidance/final_maneuver_plan
     type: carma_planning_msgs/msg/ManeuverPlan
     queue_size: 1
 
+  - topic: /guidance/plugins/pure_pursuit/plan_trajectory
+    type: carma_planning_msgs/msg/TrajectoryPlan
+    queue_size: 10
```

Comment:
Why are we keeping the last 10 messages?

Reply:
No reason, I missed this let me change